doc/source/python-api.rst

```diff
@@ -115,23 +115,26 @@ Array
 Dense Array
 -----------
 
-.. autoclass:: tiledb.DenseArray
+.. autoclass:: tiledb.libtiledb.DenseArrayImpl
```

Comment:
Rather than importing the internal class `DenseArrayImpl` (which will be confusing) we can use this technique to make sphinx use the documentation from the source of the alias (but keep the aliased name): https://stackoverflow.com/questions/38765577/overriding-sphinx-autodoc-alias-of-for-import-of-private-class#comment115374850_58982001

Reply:
Modifying https://github.com/TileDB-Inc/TileDB-Py/blob/7b5a4004f0ea6c1ec495e6aa15e58fb02726e6b1/tiledb/__init__.py#L106-L107 to:

```
from .libtiledb import DenseArrayImpl as DenseArray
DenseArray.__module__ = __name__
DenseArray.__name__ = 'DenseArray'
from .libtiledb import SparseArrayImpl as SparseArray
SparseArray.__module__ = __name__
SparseArray.__name__ = 'SparseArray'
```

gives `TypeError: cannot set '__module__' attribute of immutable type 'tiledb.libtiledb.DenseArrayImpl'`.

Having the same problem with all the similar methods I found online.
I also found no way to disable aliases for the whole project.
Any ideas?

Reply:
Try in conf.py ([like this](https://github.com/shap/shap/blob/6af9e1008702fb0fab939bf2154bbf93dfe84a16/docs/conf.py#L380-L394) from that comment)

Reply:
Getting the same error.

Reply:
Doesn't work because classes generated by Cython are immutable.
Solve it the way that works for cloud, by adding a wrapper class with basically the same name.